sdk/python/tests/components/_contextmanagers.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from contextlib import contextmanager
+
+@contextmanager
+def resolve_container_task_context():
```

Comment:
Are you going to expose this API to user? Could you give me some context about this API?

Reply:
I do not plan to do so. This is mainly useful for testing. That's why it's in the tests directory.

Reply:
Oh, I didn't see it's in test. Thanks.